src/client/layouts/error.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <page-layout v-if="error.statusCode === 404" :page="notfound" />
+  </div>
+</template>
+
+<script>
+  import notfoundI18n from '../static/data/404.json'
+  import { PageLayout } from '../components/'
+
+  export default {
```

Comment:
set `title` in `meta` to `page.title`: https://github.com/nuxt/nuxt.js/blob/master/lib/app/components/nuxt-error.vue#L27